apps/apple/FluentUITester/macos/Podfile

```diff
@@ -1,35 +1,41 @@
-# require_relative '../node_modules/@react-native-community/cli-platform-ios/native_modules'
+require_relative '../node_modules/@react-native-community/cli-platform-ios/native_modules'
 
 abstract_target 'Shared' do
-  # use_native_modules!
```

Comment:
Can macos still not find native_modules even with the include above succeeding?

Reply:
The `require relative` and `use_native_modules` are now uncommented, and seem to be working fine =)